cluster_follow.go

```diff
@@ -31,16 +34,16 @@ type walEntry struct {
 }
 
 type followSpec struct {
-	followerID int
+	followerID common.FollowerID
```

Comment:
The main change to this file is that followers are now identified by their ID, which is a combination of the server ID and the partition number.